sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/TestSuiteBase.java

```diff
@@ -233,6 +235,10 @@ protected static void truncateCollection(CosmosAsyncContainer cosmosContainer) {
         logger.info("Truncating collection {} ...", cosmosContainerId);
         List<String> paths = cosmosContainerProperties.getPartitionKeyDefinition().getPaths();
         CosmosQueryRequestOptions options = new CosmosQueryRequestOptions();
+        options.setCosmosEndToEndOperationLatencyPolicyConfig(
```

Comment:
Just curious - we need these changes because of the global copy of `CosmosClientBuilder`?

Reply:
Because to test the new override the e2e policy has to be applied to client level - and then it can have side effects because the client instance is often used for an entire test class.